### DISCUSSION TITLE
Build tag for Build -> Core -> UserOrg

### DISCUSSION THREAD

**Discussion by daveanvit** [Sunbird-Ed/Community#117] [Installation] [answered]
Kindly suggest build tag for Build -> Core -> UserOrg?
could not find it in [documentation](http://docs.sunbird.org/latest/developer-docs/server-installation/build-and-deploy/)

**Comment by sharathkashyap** [comment, marked as answer]
@daveanvit Could you please suggest the release you are looking for?
cc: @keshavprasadms

**Reply by beepdot** [reply]
Please don't run the jobs which are not mentioned in the docs. There are many additional jobs which are not required for installation process.
